docs/Admin/CrossTenantMailboxMigrationValidation.md

```diff
@@ -15,9 +15,12 @@ It will help you on:
 - Making sure the source mailbox object has no hold applied
 - Making sure the source mailbox object TotalDeletedItemsSize is not bigger than Target MailUser recoverable items size
 - Making sure the source mailbox object LegacyExchangeDN attribute value is present on the target MailUser object as an X500 proxyAddress, and give you the option to set it, as long as the Target MailUser is not DirSynced
+- Making sure the source mailbox object X500 addresses are also present on the target MailUser object.
```

Comment:
Was this already being done? I don't see this being done I believe in the new code.

Reply:
Yes, it was already done in the past, but wasn't listed yet:

                #Sync X500 addresses from source mailbox to target mailUser
                Write-Verbose -Message "Informational: Checking for missing X500 addresses on TARGET that are present on SOURCE mailbox"
                if ($SourceObject.EmailAddresses -like '*500:*') {
                    Write-Verbose -Message "SOURCE mailbox contains X500 addresses, checking if they're present on the TARGET MailUser"
                    foreach ($Address in ($SourceObject.EmailAddresses | Where-Object { $_ -like '*500:*' })) {
                        if ($TargetObject.EmailAddresses -notcontains $Address) {
                            if (!$TargetObject.IsDirSynced) {
                                Write-Host ">> Error:"$Address" is not present on the TARGET MailUser, would you like to add it? (Y/N): " -ForegroundColor Red -NoNewline
                                $AddX500 = Read-Host
                                Write-Host " Your input: "$AddX500
                                if ($AddX500.ToLower() -eq "y") {
                                    Write-Host "Informational: Adding the X500 Address"$Address" on the TARGET object"
                                    Set-TargetMailUser $TargetIdentity -EmailAddresses @{Add = $Address }
                                    #Reload TARGET object into variable as it has been changed
                                    $TargetObject = Get-TargetMailUser $TargetIdentity
                                }
                            } else {
                                Write-Host ">> Error:"$Address" is not present on the TARGET MailUser and the object is DirSynced. This is not a change that can be done directly on EXO, please add the X500 address from on-premises and perform an AADConnect delta sync" -ForegroundColor Red
                            }
                        } else {
                            Write-Host ">> Informational: The X500 address from SOURCE object is present on TARGET object" -ForegroundColor Green
                        }
                    }
                } else {
                    Write-Verbose -Message "Informational: SOURCE mailbox doesn't contain any X500 address"
                }